src/index.js

```diff
@@ -1,11 +1,24 @@
+/* eslint-disable */
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import './index.css';
 import App from './App';
+import Home from './pages/Home';
+import Calculator from './components/Calculator';
```

Comment:
- [ ] At the moment, the app in unable to start because these modules are imported wrongly. Please kindly correct the path so they can be properly imported. For example:
import Navigation from './Components/Navigation